.github/workflows/initrepo.yml

```diff
@@ -22,26 +22,32 @@ jobs:
         with:
           go-version: '^1.17.5' # The Go version to download (if necessary) and use.
       - run: go version
+      - name: Sanitize repo owner
+        uses: actions/github-script@v4
+        id: repo_slug
+        with:
+          result-encoding: string
+          script: return '${{ github.repository_owner }}'.toLowerCase()
       - name: Update the image name in .ci_env, helm chart and skaffold
         run: |
           ESCAPED_ORIGINAL=$(printf '%s\n' "$ORIGINAL_IMAGE_NAME" | tr '[:upper:]' '[:lower:]' | sed -e 's/[\/&]/\\&/g')
           ESCAPED_REPLACE=$(printf '%s\n' "$NEW_IMAGE_NAME" | tr '[:upper:]' '[:lower:]' | sed -e 's/[\/&]/\\&/g') 
           sed -i "s/${ESCAPED_ORIGINAL}/${ESCAPED_REPLACE}/g" chart/values.yaml skaffold.yaml chart/README.md
-          NEW_SERVICE_NAME=$(echo ${{ github.repository }} | sed -e 's/${{ github.repository_owner }}\///')
+          NEW_SERVICE_NAME=$(echo ${{ github.repository }} | sed -e 's/${{ steps.repo_slug.outputs.result }}\///')
```

Comment:
Careful, this is not what you intend to do here. 

Example: `github_repository=YourName/some-service` , however `github_repository_owner=YourName`. With `sed -e ' s/...` we are trying to remove the `YourName/` part of the string, so here we don't need the lower case slug.

Reply:
Sorry, I overlooked that. It should be fixed now!